src/electron-main.ts

```diff
@@ -322,7 +350,10 @@ if (global.store.get("disableHardwareAcceleration", false) === true) {
 }
 
 app.on("ready", async () => {
+    let asarPath: string;
+
     try {
+        asarPath = await getAsarPath();
```

Comment:
given that setupGlobals will do this, any point in doing it again here?

Reply:
You asked for `getAsarPath` to be the accessor, if we don't call this then how do we get at `asarPath`? It's all de-duplicated via the promise anyway.

Reply:
this function doesn't use `asarPath` though (I think)?

Reply:
It does, further down. 

![image](https://user-images.githubusercontent.com/2403652/225577305-24866e6e-9993-4f42-a428-eef3efa3fcbc.png)

Reply:
oh, my bad, apologies!